redis/client.py

```diff
@@ -2996,12 +2996,22 @@ def hlen(self, name):
         "Return the number of elements in hash ``name``"
         return self.execute_command('HLEN', name)
 
-    def hset(self, name, key, value):
+    def hset(self, name, key=None, value=None, mapping=None):
         """
-        Set ``key`` to ``value`` within hash ``name``
-        Returns 1 if HSET created a new field, otherwise 0
+        Set ``key`` to ``value`` within hash ``name``,
+        Use kwargs to set multiple key/value paris for a hash ``name``.
```

Comment:
```suggestion
        Use kwargs to set multiple key/value pairs for a hash ``name``.
```

You probably want to update this comment to reflect the use of `mapping` rather than kwargs.

Reply:
Thanks, I updated.